guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -378,6 +472,22 @@
 
     },
 
+    "PLAYER" : {
+
+        "ACTION_CANCEL"       : "@:APP.ACTION_CANCEL",
+        "ACTION_PAUSE"        : "@:APP.ACTION_PAUSE",
+        "ACTION_PLAY"         : "@:APP.ACTION_PLAY",
```

Comment:
As with other sections, I do not believe this need to be directly included.

Reply:
Removed all the @APP lines